### PR TITLE
Added mode for sharing gold across team

### PR DESCRIPTION
**Pull request by theQuazz**

### CODE REVIEW

**Comment by Marsunpaisti** [comment]
Fixed & tested

File: wurst/systems/modes/TeamGoldSharing.wurst [hunk 1]
```diff
@@ -27,7 +26,6 @@ function syncTribesToLowestGold()
             let lowestLumber = members.getFirst().getLumber()
             for m in members
                 m.setLumber(lowestLumber)
-            destroy members
```
Comment (jonathanmclaus) [inline wurst/systems/modes/TeamGoldSharing.wurst]: Why did you remove the destroy?
